crates/swc_ecma_transforms_base/src/resolver/mod.rs

```diff
@@ -488,6 +488,19 @@ impl<'a> VisitMut for Resolver<'a> {
         }
     }
 
+    fn visit_mut_switch_stmt(&mut self, s: &mut SwitchStmt) {
+        s.discriminant.visit_mut_with(self);
+
+        let child_mark = Mark::fresh(Mark::root());
+
+        let mut child_folder = Resolver::new(
+            Scope::new(ScopeKind::Block, child_mark, Some(&self.current)),
+            self.handle_types,
+        );
+
+        s.visit_mut_children_with(&mut child_folder);
```

Comment:
I think this should be `s.cases.visit_mut_with(&mut child_folder);`

Reply:
oh, thanks